### DISCUSSION TITLE
1.3.2

### DISCUSSION THREAD

**Discussion by JLyne** [JLyne/LiveAtlas#52] [General]
A quick release to fix some important issues that appeared in 1.3.0 and were not fully resolved in 1.3.1.

##Fixed
- Fixed bug causing setups using dynmap's `standalone/config.js` file to not load correctly.
- Fixed players button title
- Fixed some UI elements having the configured background colour applied multiple times, which would cause inconsistent opacity when the background colour is semi-transparent.
- Removed unintentionally added test server configurations from index.html

<hr /><em>This discussion was created from the release <a href='https://github.com/JLyne/LiveAtlas/releases/tag/v1.3.2'>1.3.2</a>.</em>

**Comment by Minocula** [comment]
Hey dear James, don't drink the donated 6 coffees all at once :D I have a general question. My map list in LiveAtlas is huge. I saw in the presentation link that its possible to create map groups. Where do I have to set these up? in the Dynmap Config or in LiveAtlas? Kind regards and thx for your great work!

**Reply by Minocula** [reply]
@JLyne Did you saw that question too? Kind regards
![2021-07-29 09_09_37-Kalender - david schell@swing info - Outlook](https://user-images.githubusercontent.com/63090608/127447674-12bd14fe-3365-4538-a83b-54ad82860efc.png)

**Comment by Minocula** [comment]
By the way....I found something strange in that version. If I share a Location Link to a person - and that person open's the URL for the first time (no visit or cache from liveatlas before) the correct map opens but with no coordinates. If I retype the URL the location URL works correctly. Was that already before of did I just noticed it now?

You can test this URL: (copy link first)
https://minecraft.trialanderror.tech/dynmap/#1;surface;-298,64,215;4

![location link](https://user-images.githubusercontent.com/63090608/127191021-62c9335b-d186-48f3-b1be-1492f45c9f9f.jpg)

**Reply by JLyne** [reply]
Hmm yeah there does seem to be an issue here. Looks like the zoom level in the URL is being ignored. Will create an issue to look at when I can.

**Reply by Minocula** [reply]
Great. It's not so bad, because the correct world still opens. But as you can imagine to show a specific spot on a map this feature is just perfect!

**Reply by Minocula** [reply]
> Hmm yeah there does seem to be an issue here. Looks like the zoom level in the URL is being ignored. Will create an issue to look at when I can.

Cool it works now in the current version! thx a lot!
